.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: python
+sudo: false
+cache:
+  pip: true
+  directories:
+    - $HOME/buildout-cache
+python:
+  - 2.7
+env:
+  - PLONE_VERSION=4.3
+  - PLONE_VERSION=5.0
+before_install:
+  - sed -ie "s#travis-5.x.cfg#travis-$PLONE_VERSION.x.cfg#" travis.cfg
+  - sed -ie "s#5.0-latest#$PLONE_VERSION-latest#" buildout.cfg
+  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
+  - mkdir $HOME/.buildout
+  - echo "[buildout]" > $HOME/.buildout/default.cfg
+  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
+  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
+install:
+  - python bootstrap-buildout.py -c travis.cfg
+  - bin/buildout -Nc travis.cfg
```

Comment:
is good practice to include `bin/buildout annotate` here for debugging purposes.